src/sql/src/plan/query.rs

```diff
@@ -147,39 +147,44 @@ pub fn plan_insert_query(
         bail!("cannot insert into system table '{}'", table.name());
     }
 
+    // Normalize column names
+    let columns: Vec<_> = columns.into_iter().map(normalize::column_name).collect();
+
     // Validate target column order.
-    let mut project_key: Vec<_> = (0..desc.arity()).collect();
-    let mut ordering = Vec::with_capacity(desc.arity());
+    let mut source_types = Vec::with_capacity(columns.len());
```

Comment:
Nit: maybe `source_types` -> `column_types`? Source usually refers to an upstream source of data in this codebase.

Reply:
I chose this due to the `source: InputSource` argument of the function as it describes the types of that whereas `column_types` would hint towards the types of the table itself. What do you think? I'm fine either way

Reply:
Yeah, good point! Totally happy to have `source` here.